test/XRP/fakes/fake-web-socket-network-client.ts

```diff
@@ -200,10 +203,19 @@ export class FakeWebSocketNetworkClientResponses {
 /**
  * A fake network client which stubs network interaction.
  */
-export class FakeWebSocketNetworkClient {
+export class FakeWebSocketNetworkClient
+  implements WebSocketNetworkClientInterface {
   public constructor(
     private readonly responses: FakeWebSocketNetworkClientResponses = FakeWebSocketNetworkClientResponses.defaultSuccessfulResponses,
   ) {}
+  findRipplePath(
+    _sourceAccount: string,
+    _destinationAccount: string,
+    _destinationAmount: string | IssuedCurrency,
+    _sendMax?: string | IssuedCurrency,
+  ): Promise<RipplePathFindResponse> {
+    throw new Error('Method not implemented.')
```

Comment:
Shouldn't this have similar dummy values to everything else?

Reply:
Oh yeah, I think I let the compiler auto-generate that for me to make it happy with the fake websocket network client that implements the interface.... because I think I also had to go and specify `findRipplePath` in the interface.  Maybe this stuff should be done in the `ripple-path-find` branch.